packages/shared/src/components/cards/v1/PostCardFooter.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { CardImage, CardVideoImage } from './Card';
+import { Post, isVideoPost } from '../../../graphql/posts';
+import { cloudinary } from '../../../lib/image';
+
+interface PostCardFooterClassName {
+  image?: string;
+}
+
+type PostCardFooterProps = {
+  insaneMode: boolean;
+  openNewTab: boolean;
+  showImage: boolean;
+  post: Post;
+  className: PostCardFooterClassName;
+};
+
+export const PostCardFooter = ({
+  post,
+  showImage,
+  className,
+}: PostCardFooterProps): ReactElement => {
+  const isVideoType = isVideoPost(post);
+  const ImageComponent = isVideoType ? CardVideoImage : CardImage;
+
+  if (!showImage) {
+    return null;
+  }
+  return (
+    <>
+      {/* {!showImage && post.author && (
+        <PostAuthor
+          author={post.author}
+          className={classNames(
+            'mx-4 mt-2 hidden tablet:flex laptop:hidden',
+            visibleOnGroupHover,
+          )}
+        />
+      )} */}
+      <ImageComponent
+        alt="Post Cover image"
+        src={post.image}
+        fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+        className={classNames(
+          'w-full object-cover',
+          className.image,
+          !isVideoType && 'mt-4',
+        )}
+        loading="lazy"
+        data-testid="postImage"
+        {...(isVideoType && { wrapperClassName: 'mt-4' })}
```

Comment:
Might break the app if the value is false. Wdyt?

Reply:
I don't think I get what might break here if `isVideoType` is `false` :)

Reply:
You are spreading the result of this line, meaning if false, you will be spreading the boolean false which is not an object you can iterate similar to object literal/arrays.

Reply:
We are doing the same thing in the original component. I think this part of the code is handled by react and not the typescript compiler / javascript runtime. Hence why it does not fail in production. It's actually a pretty common pattern that I've seen elsewhere as well 🤔

Reply:
Oh, if that is the case, feel free to move on. I learned something today 🚀